client/releases.go

```diff
@@ -29,6 +29,19 @@ func (c *Client) GetReleases(app string) (Releases, error) {
 	return releases, nil
 }
 
+// GetReleasesWithLimit requests releases of an app, overriding the default limit.
+func (c *Client) GetReleasesWithLimit(app string, limit int) (Releases, error) {
```

Comment:
Happy to go either way. I started going down the path you mention, but then I saw this pattern used for builds:

https://github.com/convox/rack/blob/master/client/builds.go#L40